index.js

```diff
@@ -5,14 +5,46 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "pink"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
-
+ console.log('This is what the traveler object looks like: ', traveler)
+let travelerInfo = document.querySelector('.traveler')
+let travelerName = travelerInfo.querySelector('h2')
+let travelerNickname = travelerInfo.querySelector('em')
+let travelerPhoto = travelerInfo.querySelector('img')
+travelerName.textContent = `${traveler.name}`
+travelerNickname.textContent = `${traveler.nickname}`
+travelerPhoto.src = `${traveler.photo}`
+travelerPhoto.alt = "Cairn terrier on wood floor"
 
 /***** Deliverable 4 *****/
 
+let sightingsArray = traveler.animalSightings
+sightingsArray.forEach(function (sightingObject) {
+    const newListItem = document.createElement('li')
+    const newP = document.createElement('p')
+    const newImg = document.createElement('img')
+    const newA = document.createElement('a')
+
+    newListItem.className = 'animal-sighting'
+    newListItem["data-id"] = `${sightingObject.id}`
```

Comment:
Nice guess, it makes sense! This does indeed adds a new key value pair to `newListItem`, since it is an object, but it doesn't add the attribute on the HTML element. Try this:

```suggestion
    newListItem.dataset.id = `${sightingObject.id}`
```